### PR TITLE
feat: tried new way to modify codes

### PR DESCRIPTION
**Pull request by baseplate-admin**

### CODE REVIEW

File: src/lib/components/shared/markdown.svelte [hunk 1]
```diff
@@ -73,3 +69,9 @@
         {@html html}
     {/await}
 </markdown>
+
+<style lang="postcss">
+    :global(pre) {
+        @apply flex bg-surface-400/50 md:rounded-[0.5vw] md:p-3 md:text-[0.9vw];
+    }
+</style>
```
Comment (tokitou-san) [inline src/lib/components/shared/markdown.svelte:77]: you forgot to add `unstyled` class?

Screenshot from 2023-08-10 20-44-04

Comment (baseplate-admin) [reply src/lib/components/shared/markdown.svelte:77]: Unstyled is not  a property

Comment (tokitou-san) [reply src/lib/components/shared/markdown.svelte:77]: Okay? add it in renderer?

Comment (baseplate-admin) [reply src/lib/components/shared/markdown.svelte:77]: We need to modify everything by hand

Comment (baseplate-admin) [reply src/lib/components/shared/markdown.svelte:77]: > Okay? add it in renderer?

can't. Cause we cant use escape

Comment (tokitou-san) [reply src/lib/components/shared/markdown.svelte:77]: Okay? sounds like a big plan lol, anyways this isnt working
fix it?

Comment (baseplate-admin) [reply src/lib/components/shared/markdown.svelte:77]: Change Background color?

**Pull request merged**